sql/modules/Date_Formats.sql

```diff
@@ -0,0 +1,506 @@
+CREATE TABLE date_formats (
+  language text check (language ~ '[[:alpha:]_]') NOT NULL,
+  country text check (country ~ '[[:alpha:]_]') NOT NULL,
+  format text check (format ~ '[[:graph:]_]') NOT NULL
+);
+CREATE UNIQUE INDEX date_formats_key ON date_formats(language,country);
+
+CREATE OR REPLACE FUNCTION date_formats__check() RETURNS TRIGGER AS
+$$
+BEGIN
+  IF ((SELECT COUNT (C.short_name) FROM Country C
+      WHERE C.short_name = NEW.Country) = 0) THEN
+    RAISE EXCEPTION 'Unknown country %', NEW.country;
+  END IF;
+  RETURN NEW;
+END;
+$$ LANGUAGE PLPGSQL;
+
+CREATE TRIGGER date_formats_trigger
+   BEFORE INSERT ON date_formats
+FOR EACH ROW EXECUTE PROCEDURE date_formats__check();
+
+-- See CPAN DateTime::Locale::Catalog
+INSERT INTO date_formats (language,country,format)
+VALUES

```

Comment:
Very nice! But we also want to load these only once.

So, probably the best place to put them is in the DDL file which creates the initial table.